pkgs/development/libraries/vtk/generic.nix

```diff
@@ -90,6 +90,14 @@ in stdenv.mkDerivation rec {
     sed -i 's/fprintf(output, shift)/fprintf(output, "%s", shift)/g' ./ThirdParty/libxml2/vtklibxml2/xpath.c
   '';
 
+  preFixup = ''
+    for lib in $out/lib/libvtk*.so; do
+      ln -s $lib $out/lib/"$(basename "$lib" | sed -e 's/-[[:digit:]]*.[[:digit:]]*//g')"
+    done
+
+    cp -r $out/include/vtk-*/. $out/include
```

Comment:
Maybe use symlinks instead, since we're not replacing `$out/include/vtk-$version`, or move the headers and make `$out/include/vtk-$version` into a symlink to its parent?

Reply:
Done (: